docs/AboutUs.md

```diff
@@ -1,59 +1,51 @@
----
-layout: page
-title: About Us
----
+# About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
```

Comment:
Think can change the email here to one of our email addresses

Reply:
sure I just put mine